pkg/asset/ignition/bootstrap/common.go

```diff
@@ -248,7 +248,9 @@ func (a *Common) getTemplateData(dependencies asset.Parents, bootstrapInPlace bo
 	}
 
 	registries := []sysregistriesv2.Registry{}
-	for _, group := range MergedMirrorSets(installConfig.Config.ImageContentSources) {
+	digestMirrorSources := ContentSourceToDigestMirror(installConfig.Config.ImageContentSources)
+	digestMirrorSources = append(digestMirrorSources, installConfig.Config.ImageDigestSources...)
```

Comment:
this is still merging content instead of erroring?

Reply:
fixed